examples/github-appcards/netlify/functions/project-cards.js

```diff
@@ -0,0 +1,168 @@
+/* 
+
+  This endpoint updates Miro App Card statuses when there is an update to their poisition
+  or column in GitHub (i.e. from "In Progress" to "Done")
```

Comment:
Suggestion:

"This endpoint updates Miro app card statuses when the corresponding GitHub issues change state. For example, from 'In progress' to 'Done'.""